test/org/sagebionetworks/bridge/services/AuthenticationServiceMockTest.java

```diff
@@ -151,7 +153,7 @@
 
     private Study study;
 
-    private Account account;
+    private GenericAccount account;
```

Comment:
We should avoid direct references to GenericAccount, since this bypasses our abstraction, and makes it more difficult to merge HibernateAccount and GenericAccount in the future (if that's the path we're planning on going). If there's something we need in GenericAccount outside of the AccountDao, we should probably move it to the interface.